src/index.html

```diff
@@ -5,8 +5,249 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="/" class="moyo__link">
+        <img
+          class="moyo__logo"
+          src="/images/logo.png"
+          alt="moyo-logo"
+        >
+      </a>
+      <nav class="navigation">
+        <ul class="nav__menu">
+          <li class="nav__item">
+            <a href="/" class="nav__link is-active">Apple</a>
```

Comment:
```suggestion
            <a href="#" class="nav__link is-active">Apple</a>
```
use `#` as default href

Reply:
Thanks)